src/bin/Interpret.cc

```diff
@@ -1307,7 +1309,11 @@ void Interpret::getInterpolants(const ASTNode& n)
         interpolationContext->getSingleInterpolant(itps, partitionings[0]);
     }
 
+    CounterRewriter interesting = CounterRewriter(*logic);
     for (int j = 0; j < itps.size(); j++) {
+        interesting.rewrite(itps[j]);
+        std::cout << "Amount of vars: " << interesting.config.varNumber <<
+            "\nAmount of boolean operators:" << interesting.config.boolNumber << "\n";
```

Comment:
Experimentation stuff, shouldn't go to master.

Reply:
Removed